extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthProcessor.java

```diff
@@ -220,6 +225,13 @@ public void defineHealthRoutes(BuildProducer<RouteBuildItem> routes,
                     new RouteBuildItem(health.rootPath + health.groupPath + "/" + healthGroup,
                             handler, HandlerType.BLOCKING));
         }
+
```

Comment:
I think this should be in a separate build step that means it is more easily not included if OpenAPI isn't present.

Can something like this be done: https://github.com/quarkusio/quarkus/blob/master/extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/MicrometerProcessor.java#L58

Reply:
Cool, made it a separate method.